code/modules/mob/new_player/new_player.dm

```diff
@@ -157,6 +161,15 @@
 				return 0
 
 		LateChoices()
+	if(href_list["cluwnebanned"])
+		if(!ticker || ticker.current_state <= GAME_STATE_PREGAME)
```

Comment:
due to lack of sanity, this allows you to use href exploits to join as a cluwne even if not banned